contracts/rollup/BatchManager.sol

```diff
@@ -59,6 +62,20 @@ contract BatchManager is Parameters {
         _;
     }
 
+    constructor(
```

Comment:
Solidity requires the value of immutable to be assigned immediately. We can't assign the value in the inherited contracts.